easybuild/easyconfigs/b/bwa-meth/bwa-meth-0.2.2-iccifort-2019.1.144-GCC-8.2.0-2.31.1.eb

```diff
@@ -0,0 +1,41 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonBundle'
+
+name = 'bwa-meth'
+version = '0.2.2'
+
+homepage = 'https://github.com/brentp/bwa-meth'
+description = """Fast and accurante alignment of BS-Seq reads."""
+
+toolchain = {'name': 'iccifort', 'version': '2019.1.144-GCC-8.2.0-2.31.1'}
+
+multi_deps = {'Python': ['3.7.2', '2.7.15']}
+
+dependencies = [
+    ('SAMtools', '1.9'),
+    ('BWA', '0.7.17'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('toolshed', '0.4.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/t/toolshed'],
+        'checksums': ['9541d45f14f9c9ee665289fddc2a40135a8b7abdb600acd85d6c074475ce8238'],
+    }),
+    (name, version, {
+        'source_tmpl': 'v%s.tar.gz' % version,
+        'source_urls': ['https://github.com/brentp/bwa-meth/archive'],
+        'checksums': ['b7284f016be0a99486219b272d90bda47ee28969ea7ef9f28c701b23f2a64955'],
+        'modulename': 'bwameth',
+    }),
+]
+
```

Comment:
@Darkless012 You'll need to fix the shebang in the Python script:

```
$ head -1 /software/bwa-meth/0.2.2-iccifort-2019.1.144-GCC-8.2.0-2.31.1/bin/{bwameth.py,toolshed}
==> /software/bwa-meth/0.2.2-iccifort-2019.1.144-GCC-8.2.0-2.31.1/bin/bwameth.py <==
#!/software/Python/2.7.15-GCCcore-8.2.0/bin/python

==> /software/bwa-meth/0.2.2-iccifort-2019.1.144-GCC-8.2.0-2.31.1/bin/toolshed <==
#!/software/Python/2.7.15-GCCcore-8.2.0/bin/python
```

Can we also add sanity check commands for both these tools with `--help` (which would have caught this issue)?